plugins/postcss-cascade-layers/test/nested-case-insensitive.expect.css

```diff
@@ -9,9 +9,6 @@
 		}
 	}
 
-@KEYFRAMES slide-left {
-	}
-
```

Comment:
This was the bug I wanted to fix.
I totally missed this but an empty `@KEYFRAMES` was created before this update.

This obviously breaks a lot of things.